packages/steveo/src/consumers/sqs.ts

```diff
@@ -66,15 +67,19 @@ class SqsRunner extends BaseRunner implements IRunner {
               await this.deleteMessage(c.topic, message);
             }
 
-            this.logger.debug('Start subscribe', c.topic, params);
             if (!task) {
               this.logger.error(`Unknown Task ${c.topic}`);
               return;
             }
 
-            await task.subscribe(params, runnerContext);
-            this.logger.debug('Completed subscribe', c.topic, params);
-            const completedContext = getContext(params);
+            const { context = null, ...data } = c.payload;
```

Comment:
QQ: isn't this is how we set context from the provider: https://github.com/ordermentum/steveo/blob/c412745e32bce8be3190896459ef924cc2c9f3ee/packages/steveo/src/producers/sqs.ts#L109 wouldn't it be under `_meta` attribute? Although I'm entirely sure about what context we are talking here so I'll put this as a clarification question

Reply:
job helper used to pass a context about the job to the task. for example https://github.com/ordermentum/ordermentum/blob/develop/src/jobs/tasks/sftp_report_task.ts#L36

Reply:
@sugendran Ah, I remember I came past through this, when the job dispatch the message: https://github.com/ordermentum/steveo/blob/c412745e32bce8be3190896459ef924cc2c9f3ee/packages/scheduler-prisma/src/helpers.ts#L113, got it